code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -155,10 +155,10 @@
 	armrest = GetArmrest()
 	armrest.layer = ABOVE_MOB_LAYER
 	return ..()
-
+/* Commented out due to duplication conflicts with the sunset modularized version
```

Comment:
dont comment out code if its not needed

Reply:
It was commented out because it uses

>icons/obj/objects.dmi 

With the bed rest stuff, which I had compiling issues about it duplicates.